src/serializer/ResidualHeapSerializer.js

```diff
@@ -897,6 +899,7 @@ export class ResidualHeapSerializer {
           this._assignProperty(
             () => t.memberExpression(this.getSerializeObjectIdentifier(val), t.identifier("length")),
             () => {
+              // $FlowFixMe flow thinks lenProperty is undefined :(
```

Comment:
Could this be because Flow isn't sure if it can be reassigned? Would using `const` help?

Reply:
Const should help.